pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -0,0 +1,160 @@
+package release_payload_controller
+
+import (
+	"context"
+	"fmt"
+	releasev1alpha1 "github.com/openshift/release-controller/pkg/client/clientset/versioned/typed/release/v1alpha1"
+	releasepayloadinformer "github.com/openshift/release-controller/pkg/client/informers/externalversions/release/v1alpha1"
+	releasepayloadlister "github.com/openshift/release-controller/pkg/client/listers/release/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+type PayloadAcceptedController struct {
+	kubeClient kubernetes.Interface
```

Comment:
in here we specify the smaller clients.  I'd like to see things like jobClient, configmapClient, prowjobClient, releasePayloadClient, etc